src/index.js

```diff
@@ -29,12 +26,28 @@ const defaultOptions = {
 
 const SNAPSHOT_TITLE = 'Snapshot Diff:\n';
 
+const identity = value => value;
+let serializers = [reactSerializer];
```

Comment:
Please export this property, so that users are able to import all the default serializers when calling `setSerializers`